src/main/java/com/amplitude/exception/AmplitudeException.java

```diff
@@ -0,0 +1,7 @@
+package com.amplitude.exception;
+
+public class AmplitudeException extends Exception {
+    public AmplitudeException(String message) {
+        super(message);
+    }
+}
```

Comment:
new line, I guess we can run formatter later to cover this. Or you can config the plugin to run on every save

Reply:
Sounds good, will check how to make the formatter run for every save, that's much more convenient.